ads/google/doubleclick.js

```diff
@@ -53,6 +54,18 @@ export function doubleclick(global, data) {
     };
   }
 
+  // Center the ad in the container.
+  const container = global.document.querySelector('#c');
+  if (container) {
```

Comment:
Do you mean to add `dev().assertElement(container)` in addition to checking `if(container) { ...`?